substrate/bin/node/runtime/src/lib.rs

```diff
@@ -1391,7 +1391,7 @@ parameter_types! {
 	pub AllowMultiAssetPools: bool = true;
 	pub const PoolSetupFee: Balance = 1 * DOLLARS; // should be more or equal to the existential deposit
 	pub const MintMinLiquidity: Balance = 100;  // 100 is good enough when the main currency has 10-12 decimals.
-	pub const LiquidityWithdrawalFee: Permill = Permill::from_percent(0);  // should be non-zero if AllowMultiAssetPools is true, otherwise can be zero.
+	pub const LiquidityWithdrawalFee: Permill = Permill::from_parts(0);
```

Comment:
But... it says it should be non zero. Whats the tradeoff we are making by defying this ?

Reply:
1. This comments gets removed in newer versions of substrate
2. I didn't find any explanation on what author had in mind.
3. I'm confident that there's no technical reason behind it - it's an economical one. With MultiAssetPools the liquidity will be much more divided. Having fees for withdrawals makes it more likely that people will keep their liquidity in given pool instead of constantly moving it around.